### PR TITLE
Pushjet support

### PR DESCRIPTION
**Pull request by Mechazawa**
This branch adds support for Pushjet. Pushjet is currently a WIP project with it's main focus being the lack of registration and the fact that you can self-host it.

### CODE REVIEW

File: push.cpp [hunk 1]
```diff
@@ -647,6 +647,31 @@ class CPushMod : public CModule
...
+                if (message_uri != "")
+                {
+                    params["link"] = message_uri;
+                }
+                
+                if (options["message_priority"] != "")
+                {
+                    params["level"] = options["message_priority"];
+                }
+            }
```
Comment (jreese) [inline push.cpp]: Can you fix the indenting to use tabs to be consistent with the rest of the file?

File: push.cpp [hunk 2]
```diff
@@ -1382,6 +1407,10 @@ class CPushMod : public CModule
 						{
 							PutModule("Note: Slack requires setting 'secret' (from webhook) and 'target' (channel or username), optional 'username' (bot name)");
 						}
+                        else if (value == "pushjet")
+                        {
+                            PutModule("Note: Pushjet requires setting 'secret' (service key) option");
+                        }
```
Comment (jreese) [inline push.cpp]: Same

**Comment by jreese** [comment]
Sorry for the delay in reviewing.  Only thing I'd like to see is changing spaces to tabs for consistency.  Otherwise looks good.

**Comment by Mechazawa** [comment]
Does 8e3c300 fix the issue?

**Comment by jreese** [comment]
Sorry, missed your update until you commented.  Github says there's a merge conflict now, so I'll try rebasing it on my end.

**Comment by jreese** [comment]
Fixed the merge conflict and pushed to master.  Thanks!

**Pull request closed without merging**